jwql/website/apps/jwql/templates/base.html

```diff
@@ -131,6 +131,9 @@
 		      		{% endfor %}
 		      		<li class="nav-item active">
 	            		<a class="nav-link" href={{ url('jwql:edb') }}>EDB<span class="sr-only">(current)</span></a>
+					  </li>
+					<li class="nav-item active">
+	            		<a class="nav-link" href={{ url('jwql:table_viewer') }}>TABLE VIEWER<span class="sr-only">(current)</span></a>
```

Comment:
While "table viewer" makes a lot of sense here, the real estate in the nav bar is getting pretty small.  I wonder if we should just name this link "JWQLDB" instead (this would also be consistent with the "EDB" link).

Reply:
I like this idea. I can make the change!